base/src/main/java/org/mozilla/jss/ssl/javax/JSSSocket.java

```diff
@@ -22,13 +22,13 @@
 import javax.net.ssl.SSLContext;
 import javax.net.ssl.SSLParameters;
 import javax.net.ssl.SSLSession;
-import javax.net.ssl.SSLSocket;
 import javax.net.ssl.X509KeyManager;
 import javax.net.ssl.X509TrustManager;
 
 import org.mozilla.jss.pkcs11.PK11Cert;
 import org.mozilla.jss.pkcs11.PK11PrivKey;
 import org.mozilla.jss.provider.javax.crypto.JSSTrustManager;
+import org.mozilla.jss.ssl.SSLSocket;
```

Comment:
This is needed to simplify the transition by minimizing the required changes to existing code. Basically any code that creates the old `SSLSocket` can be changed to create `JSSSocket` instead, but the code that uses the `SSLSocket` instance can automatically use the `JSSSocket` instance without any changes since it's a subclass, and `JSSSocket` will inherit the constants defined in `SSLSocket` too.

Reply:
The changes will be needed in order to remove SSLSocket but we can do later.